chapter3/tests/t11/php-fpm.conf

```diff
@@ -0,0 +1,38 @@
+[global]
+pid = /run/php-fpm/php-fpm.pid
+
+error_log = /dev/stderr
+log_level = error
+
+daemonize = no
+events.mechanism = epoll
+
+[www]
+user = nginx
+group = nginx
+listen = /run/php-fpm/www.sock
+listen.owner = nginx
+listen.group = nginx
+
+pm = static
+pm.max_children = 1
+
+access.log = /dev/stderr
+access.format = "%R - %u %t \"%m %r%Q%q\" %s %f %{mili}d %{kilo}M %C%%"
+request_terminate_timeout = 20s
+catch_workers_output = yes
+
+[www-slow]
+user = nginx
+group = nginx
+listen = /run/php-fpm/www-slow.sock
+listen.owner = nginx
+listen.group = nginx
+
+pm = static
+pm.max_children = 1
+
+access.log = /dev/stderr
+access.format = "%R - %u %t \"%m %r%Q%q\" %s %f %{mili}d %{kilo}M %C%%"
+request_terminate_timeout = 11m ; The report page is really slow!
```

Comment:
not a good thing as it will kill the worker
see solution around max_execution_time

Reply:
For slow report: `max_execution_time` = 10 min 30 sec, `request_terminate_timeout` = 11 min
Global: `max_execution_time` = 30sec
According to the docs these two options should combine: https://www.php.net/manual/en/install.fpm.configuration.php